<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAnnonce</class>
 <widget class="QDialog" name="DialogAnnonce">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="l_nbPhotos">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>540</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>x photos disponibles</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_retour">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>560</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>511</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Offre disponible depuis le </string>
   </property>
  </widget>
  <widget class="QLabel" name="l_titre">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TITRE</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>671</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>377</width>
     <height>181</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>Prix</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Pi√®ces</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Surface</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Ville</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Adresse</string>
    </property>
   </row>
  </widget>
  <widget class="QPushButton" name="b_typeAnnonce">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>560</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Vendu</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_retour_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>560</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QWidget" name="photoPrincipale" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
   <widget class="QLabel" name="l_photo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="photoSupp">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>331</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b_next">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_previous">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
